echo-bot.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3.10
+
+import logging
+import json
+from telegram import Update, ForceReply
+from telegram.ext import (
```

Comment:
Not a big fan of such imports but ok

Reply:
This is the requirement of the "black" linter.